openedx_blockstore_relay/transfer_data.py

```diff
@@ -5,23 +5,32 @@
 
 import json
 import logging
-import os
 
-import requests
-from django.conf import settings
 from django.utils.translation import gettext as _
-from future.moves.urllib.parse import urljoin
-from lxml.etree import Element
-from lxml.etree import tostring as etree_tostring
-from xblock.core import XML_NAMESPACES
 
 from . import compat
-from .adapters import add_url_name_mixin, override_export_fs
+from .block_serializer import XBlockSerializer
+from .blockstore_client import add_file_to_draft, commit_draft, create_bundle, create_draft
 
-LOG = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
+BUNDLE_DRAFT_NAME = 'relay_import'
+BUNDLE_SCHEMA_VERSION = 0.1
 
 
-def transfer_to_blockstore(block_key, bundle_uuid=None, collection_uuid=None):
+def _bundle_type(block_type):
+    """
+    Return the manifest type to set for the given block_type.
+    """
+    if block_type in ['course']:
+        bundle_type = 'course'
+    elif block_type in ['chapter', 'sequential']:
+        bundle_type = 'collection'
```

Comment:
It would get confusing to use `collection` for two things (Collections and a block type).

Reply:
> It would get confusing to use collection for two things (Collections and a block type).

That was some old existing code, and I don't know if we'll even end up using this `type` field in `bundle.json`.... but you make a good point, so I've changed that word to `section` for now.